samples/modules/pmci/mctp/usb_endpoint/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(mctp_endpoint)
```

Comment:
project name could use something more descriptive, I get I'm guilty of probably starting this but perhaps mctp_usb_endpoint would be more descriptive